### PR TITLE
Update Roadmap

### PR DESCRIPTION
**Pull request by c4pt0r**
update roadmap

### CODE REVIEW

File: docs/2018-ROADMAP.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This document defines the roadmap for TiKV development.
 - [x] Raft Learner - Support Raft learner to smooth the configuration change process
 - [x] Raft Pre-vote - Support Raft pre-vote to avoid unnecessary leader election on network isolation
 - [ ] Joint Consensus - Change multi members safely
-- [ ] Multi-thread Raftstore - Process Region Raft logic in multiple threads
+- [X] Multi-thread Raftstore - Process Region Raft logic in multiple threads
 - [ ] Multi-thread Apply Pool - Apply Region Raft committed entries in multiple threads
```
Comment (ngaut) [inline docs/2018-ROADMAP.md]: I think Multi-thread Apply Pool is done too.

Comment (BusyJay) [reply docs/2018-ROADMAP.md]: Indeed.

Comment (c4pt0r) [reply docs/2018-ROADMAP.md]: Ok, will fill it

**Comment by c4pt0r** [comment]
@siddontang PTAL

**Pull request merged**